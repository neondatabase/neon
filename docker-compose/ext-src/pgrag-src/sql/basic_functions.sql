-- Basic function tests
SELECT rag.markdown_from_html('<p>Hello</p>');

SELECT array_length(rag.chunks_by_character_count('the cat sat on the mat', 10, 5), 1);
