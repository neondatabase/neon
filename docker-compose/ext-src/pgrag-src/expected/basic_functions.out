-- Basic function tests
SELECT rag.markdown_from_html('<p>Hello</p>');
 markdown_from_html 
--------------------
 Hello
(1 row)

SELECT array_length(rag.chunks_by_character_count('the cat sat on the mat', 10, 5), 1);
 array_length 
--------------
            3
(1 row)

